clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/client/TitusRegion.java

```diff
@@ -30,6 +30,7 @@
   private final String applicationName;
   private final String url;
   private final int port;
+  private final Boolean experimentalFeaturesEnabled;
```

Comment:
Can we make this into a string or list called featureFlags? 

If this was a list, then we can enabled featureFlags like `skipJobIds,useInstanceCachingAgent`, etc vs having to have it be all or nothing

Reply:
Sure, I'll make that change. That's a great idea.